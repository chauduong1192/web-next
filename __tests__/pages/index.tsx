import { shallow } from 'enzyme';
import * as React from 'react';

import Error from '../../pages/404';

describe('Home', () => {

  it('should render Index', () => {
    const wrapper = shallow(<Error />);
    expect(wrapper).toMatchSnapshot();
  });
});
