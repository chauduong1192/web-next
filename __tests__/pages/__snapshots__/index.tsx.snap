// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Index 1`] = `ShallowWrapper {}`;
